.github/CODEOWNERS

```
@@ -28,6 +28,7 @@
 /lib/cli.nix                @edolstra @nbp @Profpatsch
 /lib/debug.nix              @edolstra @nbp @Profpatsch
 /lib/asserts.nix            @edolstra @nbp @Profpatsch
+/lib/path.*                 @fricklerhandwerk
```

```suggestion
/lib/path.*                 @fricklerhandwerk @infinisil
```

You should have a say in that, too.

---

I guess it doesn't hurt to be explicit here, but I'm also already notified for all changes in `/lib` anyways

---

I'm pretty sure you're not: [the last match takes precedence](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file). Without the change, you'd only be requested as reviewer for files in `/lib` which are not explicitly listed.